src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -694,6 +705,18 @@ protected bool IsAlive
         }
 
         #region Location extraction & manipulation
+        /// <summary>
+        /// Sets an exsisting element's pinned status
+        /// </summary>
+        /// <param name="value">Value for pin status true/false</param>
+        public void SetPinnedStatus(bool value)
```

Comment:
More suggestive parameter name ?
```suggestion
        public void SetPinnedStatus(bool pinned)
```